index.html

```diff
@@ -1407,6 +1418,15 @@ <h3>The <dfn>writeMultipleProperties()</dfn> method</h3>
         </ol>
       </div>
     </section>
+    <section>
+      <h3>The <dfn>writeAllProperties()</dfn> method</h3>
+      <div>
```

Comment:
This is still left here?

Reply:
I left there to add the editor's note. Should I remove the section and move the editor's note to writeMultiple?

Reply:
We don't have `writeAll` in the Web IDL, therefore we don't need a section with that. 
Please put the ednote under the Web IDL (kind of explaining the gap there).
We could also put the writeAllProperties() signature in the WebIDL, but commented out.

Reply:
I added a note at the end of the full web IDL.